src/renderer/components/SyncView/SyncAppBar.js

```diff
@@ -29,33 +51,47 @@ const SyncAppBar = ({ onClickSelectSyncfile, onClickNewSyncfile }) => {
   // Check connection every 2 seconds
   useEffect(() => {
     const intervalCheck = setInterval(() => {
-      wifi
-        .getCurrentConnections()
-        .then((conn) => setCurrentConnection(conn && conn[0]))
+      wifi.getCurrentConnections().then((conn) => setCurrentConnection(conn && conn[0]))
```

Comment:
Can you add a `catch` for errors here? Probably ok to fail silently, or maybe add a message like "connection unknown".